plugins/task-plugin/src/task/che-task-runner.ts

```diff
@@ -68,7 +68,7 @@ export class CheTaskRunner {
                 shellArgs: ['-c', `${taskConfig.command}`],
 
                 attributes: {
-                    CHE_MACHINE_NAME: machineName,
+                    CHE_MACHINE_NAME: containerName,
```

Comment:
Does it make sense to rename `CHE_MACHINE_NAME` as well to be more consistent?

Reply:
Sure, but according to https://github.com/eclipse/che/issues/13667#issuecomment-509307894 we decided to do changes in two steps, the PR is the first step